### PR TITLE
add dependency identifiers to dictionary

### LINKED ISSUE #12: teach the tool about std APIs
For example, I'm getting warned about `ValueOf` in the following godoc line:

> such as those used as arguments to reflect.TypeOf or reflect.ValueOf.

### PR DESCRIPTION
**Pull request by kortschak**
Fixes #12.

/cc @mvdan

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -339,6 +344,13 @@ func addIdentifiers(spelling *hunspell.Spell, pkgs []*packages.Package) error {
 		for _, f := range p.Syntax {
 			ast.Walk(v, f)
 		}
+		for _, dep := range p.Imports {
```
Comment (mvdan) [inline main.go:347]: nit: I think you want Deps, as that includes indirect imports too.

Comment (kortschak) [reply main.go:347]: Sorry, where is that? I don't see a Deps field, but this will recur up the tree.

Comment (mvdan) [reply main.go:347]: Ah, my bad; it's a field in `go list -json`, but it's omitted from `go/packages`. Imports sounds good if you're going to recurse yourself.

**Review by mvdan — APPROVED** [review]
SGTM, I imagine you'd want a test :)

**Comment by kortschak** [comment]
Test added.

**Pull request merged**